definitions/checks/foreman_login_session.rb

```diff
@@ -0,0 +1,21 @@
+class Checks::ForemanLoginSession < ForemanMaintain::Check
+  metadata do
+    label :foreman_login_session
+    tags :foreman_login_session
```

Comment:
the `tags` define when the check should be run. I guess we want to run this as part of the `health check` command and also with the `pre_upgrade` checks. For that  the tags should be set to `:default`.

Reply:
Thanks, I will change the tag to `:default`